lib/Pipeline.js

```diff
@@ -82,14 +83,22 @@ Pipeline.processJSON = function(gltf, options) {
  */
 Pipeline.processJSONWithExtras  = function(gltfWithExtras, options) {
     options = defaultValue(options, {});
+
+    addDefaults(gltfWithExtras, options);
+
+    // Generate stats for unoptimized input
+    if (options.stats) {
+        gltfStatistics.printStatistics(gltfWithExtras, options.inputPath);
+    }
```

Comment:
Right, and per our offline discussion, gltfStatistics no longer depends on `addDefaults`. So it would be ok if we decide to move `addDefaults` back to its original location.